nav2_bringup/launch/map_server_params.yaml

```diff
@@ -0,0 +1,8 @@
+map_server:
+    ros__parameters:
+        yaml_filename: "test_map.yaml"
+        map_type: "occupancy"
+        free_thresh: 0.196
```

Comment:
As I look at the various map files we have, I don't see us wanting to tweak these values for all maps, but rather tweak them per pgm. For instance, I can't see a use case where we'd take a PGM and interpret it as trinary in one case and scale or raw in another.

Likewise, I can see you maybe tweaking the free_thresh for a particular map, but once you've got it right for that map, you'd leave it alone. Another map collected on a different day by a different mechanism might get different values.

I think all these parameters should be kept in the map.yaml file. Maybe you could override them by providing them as parameters to the map server like you have, but I see that as the exceptional use case.